lint-staged.config.js

```diff
@@ -0,0 +1,10 @@
+function globForCode(/** @type {string} */ toolName) {
+  return '*.?(c|m){js,ts}?(x)' + `*(${toolName})`;
```

Comment:
What's the point of this? To have multiple tasks for the same effective pattern?

Sorry, I'm not familiar with `lint-staged` configuration.

Reply:
Yes. If it's done this way, the tasks are ran in parallel.

Reply:
And you want them to be separate because you're passing filenames to one (`eslint`) but not the other (`tsc`), so you can't use an array?

Reply:
All right. Looks weird, but whatever.

Reply:
An array of tasks are ran in serial. We can use an array but that means no parallel execution for speedup.